mcs_benchmark_data/_submission_pipeline.py

```diff
@@ -0,0 +1,10 @@
+from mcs_benchmark_data._pipeline import _Pipeline
+
+
+class _SubmissionPipeline(_Pipeline):

```

Comment:
You have a `_BenchmarkSubmissionTransformer` but a `SubmissionPipeline`. The ontology class is `Submission`, so I'd probably use that. `_SubmissionTransformer`.